src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,91 @@
+package duke.storage;
+
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.parser.Parser;
+import duke.task.ToDo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    private String filePath;
+
+    public Storage() {
+        this.filePath = "./data/tasks.txt";
+    }
+
+    public ArrayList<Task> load() throws DukeException {
```

Comment:
it will be helpful to know what load is loading too

```suggestion
public ArrayList<Task> loadSave() throws DukeException {
```